2023-06-11 12:31:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop table if exists [*]user cascade "; expected "identifier"; SQL statement:
drop table if exists user cascade  [42001-214]
2023-06-11 12:31:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table [*]user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id)) [42001-214]
2023-06-11 12:32:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop table if exists [*]user cascade "; expected "identifier"; SQL statement:
drop table if exists user cascade  [42001-214]
2023-06-11 12:32:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table [*]user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id)) [42001-214]
2023-06-11 12:33:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop table if exists [*]user cascade "; expected "identifier"; SQL statement:
drop table if exists user cascade  [42001-214]
2023-06-11 12:33:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table [*]user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id)) [42001-214]
2023-06-11 12:33:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop table if exists [*]user cascade "; expected "identifier"; SQL statement:
drop table if exists user cascade  [42001-214]
2023-06-11 12:33:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table [*]user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table user (id bigint not null, email varchar(255) unique, password varchar(255), primary key (id)) [42001-214]
2023-06-11 12:34:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table [*]user (id bigint not null, email varchar(255), password varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table user (id bigint not null, email varchar(255), password varchar(255), primary key (id)) [42001-214]
2023-06-11 12:34:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table if exists [*]user drop constraint if exists UK_ob8kqyqqgmefl0aco34akdtpe"; expected "identifier"; SQL statement:
alter table if exists user drop constraint if exists UK_ob8kqyqqgmefl0aco34akdtpe [42001-214]
2023-06-11 12:34:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "alter table if exists [*]user add constraint UK_ob8kqyqqgmefl0aco34akdtpe unique (email)"; expected "identifier"; SQL statement:
alter table if exists user add constraint UK_ob8kqyqqgmefl0aco34akdtpe unique (email) [42001-214]
2023-06-11 17:17:38 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'244,90' (PRODUCT_ITEM: ""INITIAL_PRICE"" NUMERIC(38, 2))"; SQL statement:
INSERT INTO "PUBLIC"."PRODUCT_ITEM"("ITEM_URL","INITIAL_PRICE","NEW_PRICE","PICTURE_URL","PRODUCT_PRODUCT_CODE","WEBSITE_NAME")VALUES(?,?,?,?,?,?) [22018-214]
2023-06-11 17:17:38 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'244,90' (PRODUCT_ITEM: ""INITIAL_PRICE"" NUMERIC(38, 2))"; SQL statement:
INSERT INTO "PUBLIC"."PRODUCT_ITEM"("ITEM_URL","INITIAL_PRICE","NEW_PRICE","PICTURE_URL","PRODUCT_PRODUCT_CODE","WEBSITE_NAME")VALUES(?,?,?,?,?,?) [22018-214]
2023-06-18 10:26:35 jdbc[55]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2023-06-18 11:25:34 jdbc[76]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
